src/main/java/io/kestra/plugin/azure/batch/job/Create.java

```diff
@@ -119,6 +120,15 @@
                           imageName: python
                 """
         )
+    },
+    metrics = {
+        @Metric(name = "io.read.ops.count", type = "io.kestra.core.models.executions.metrics.Counter", description = "The number of read I/O operations performed by the task."),
```

Comment:
Use the type constant. Same for `Timer.TYPE` 🙏 

```suggestion
        @Metric(name = "io.read.ops.count", type = Counter.TYPE, description = "The number of read I/O operations performed by the task."),
```

Reply:
Does this change apply to only this file or every plugin file?

Reply:
For every file ;-)